lib/src/utils/markdown.dart

```diff
@@ -217,13 +228,15 @@ String markdown(
     ],
     inlineSyntaxes: [
       StrikethroughSyntax(),
+      MultipleLinebreaksSyntax(),
       LinebreakSyntax(),
       SpoilerSyntax(),
       EmoteSyntax(getEmotePacks),
       PillSyntax(),
       MentionSyntax(getMention),
       InlineLatexSyntax(),
     ],
+    inlineOnly : true,
```

Comment:
why need this property?

Reply:
to test the whole string without splitting